docs/site/Using-TypeORM-with-LoopBack.md

```diff
@@ -0,0 +1,134 @@
+---
+lang: en
+title: 'How to use TypeORM with LoopBack'
+keywords: LoopBack 4.0, LoopBack 4, TypeORM
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Using-typeorm-with-loopback.html
+---
+
+## Overview
```

Comment:
Perhaps we can use the README.md content directly instead of duplicating the content? 
See the [JWT authentication extension docs](https://github.com/strongloop/loopback-next/blob/master/docs/site/JWT-authentication-extension.md ) as an example.